ArkGameExample/TankRaceGame/TankRaceGameCollisionStrategyManager.swift

```diff
@@ -14,19 +14,19 @@ class TankRaceGameCollisionStrategyManager {
     private var strategies: [UInt32: [UInt32: TankRaceCollisionHandlingStrategy]] = [:]
 
     init() {
-//        register(strategy: BallWallCollisionStrategy(),
-//                 for: (TankGamePhysicsCategory.ball, TankGamePhysicsCategory.wall))
+        register(strategy: TankRaceBallWallCollisionStrategy(),
+                 for: (TankGamePhysicsCategory.ball, TankGamePhysicsCategory.wall))
         register(strategy: TankRaceBallRockCollisionStrategy(),
                  for: (TankGamePhysicsCategory.ball, TankGamePhysicsCategory.rock))
-//        register(strategy: TankWaterCollisionStrategy(),
-//                 for: (TankGamePhysicsCategory.tank, TankGamePhysicsCategory.water))
+        register(strategy: TankFinishLineCollisionStrategy(),
+                 for: (TankGamePhysicsCategory.tank, TankGamePhysicsCategory.water))
```

Comment:
we can refactor after demo today to have a separate category for each game so there's not too much confusion, or push these to a `shared-lib` with a more general name like `TankGamePhysicsCategory.damp`